README.md

```diff
@@ -8,13 +8,14 @@
 ```
 git clone [your fork]
 cd indyhackers.org
-bundle install
-bundle exec rake db:setup
+bin/setup
```

Comment:
Are we OK just referencing the bin/setup script here?

Reply:
I don't see why not, as long as bin/setup works correctly (and I assume it does since you worked on it)(

Reply:
Awesome! After adjusting the bin/setup, I used it to rebuild my environment locally and it worked 🎉